VoucherifySwiftSdk.podspec

```diff
@@ -1,7 +1,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'VoucherifySwiftSdk'
-  s.version          = '4.0.0'
+  s.version          = '4.1.0'
```

Comment:
Did you already push a new version in Cocoapods repository?

Reply:
nope, just code changes

Reply:
We can think of releasing a new version if needed (e.g after merging to master). Just to avoid potential confusion that v4.1.0 doesn't exist in Cocoapods repository.

Reply:
ok 👍 reverted all changes related to version, leaving only changes in the logic